internal/service/device.go

```diff
@@ -47,19 +47,29 @@ func (h *ServiceHandler) ListDevices(ctx context.Context, request server.ListDev
 	if request.Params.LabelSelector != nil {
 		labelSelector = *request.Params.LabelSelector
 	}
+	fieldSelector := ""
+	if request.Params.FieldSelector != nil {
+		fieldSelector = *request.Params.FieldSelector
+	}
 
 	labelMap, err := labels.ConvertSelectorToLabelsMap(labelSelector)
 	if err != nil {
 		return server.ListDevices400JSONResponse{Message: err.Error()}, nil
 	}
 
+	fieldMap, err := labels.ConvertSelectorToLabelsMap(fieldSelector)
```

Comment:
this function splits field selector by comma which breaks querying for multiple values.
If I try to do 
```
/devices?fieldSelector=applications.summary.status=Healthy,Unknown
```

we split it into two selectors
`applications.summary.status=Healthy` and `Unknown`.
The second one is invalid

Reply:
Yeah this took a little bit of thought thanks for the report this should be resolved

```
devices?statusFilter=updated.status=UpToDate,Unknown&statusFilter=applications.summary.status=Unknown
```

Reply:
works fine